Runtime/Shaders/ComputeShaderHelper.cs

```diff
@@ -29,7 +30,8 @@ internal class ComputeShaderHelper {
             renderTarget.height != maxTextureSize.y) {
             if (renderTarget != null) renderTarget.Release();
 
-            renderTarget = new RenderTexture(maxTextureSize.x, maxTextureSize.y, 32) {
+            // We don't want sRGB here!
+            renderTarget = new RenderTexture(maxTextureSize.x, maxTextureSize.y, 32, DefaultFormat.HDR) {
```

Comment:
Is this supported by all rendering pipelines?

Reply:
Also, would that render correctly when creating a build for mobile?

Reply:
Yeah, this is just needed so the baked maps are in the correct format. Without this, they are in sRGB which is not correct, especially noticeable when baking smoothness to albedo alpha.

Reply:
Do you need HDR, or a format in linear space ? They are quite different, and the HDR one consumes much more memory.

I would recommend to use the RenderTexture constructor that has `GraphicsFormat` type so that we are sure what we want to use.